File: python_skills/pewee_orm/model/student.py
# encoding=utf-8
import peewee

db = peewee.MySQLDatabase("my_data", host="localhost", user="root", passwd="111111")


class MySQLModel(peewee.Model):
    class Meta:
        database = db


class Student(MySQLModel):
    name = peewee.CharField(primary_key=True)
    password = peewee.CharField()
    address = peewee.CharField()
    score = peewee.IntegerField()

db.connect()

File: gogoing/src/model/student.go
package model

import (
	"fmt"
)

/*
bean
*/
type Student struct {
	name     string
	password string
}

func (s *Student) NewStudent(name, password string) {
	s.name = name
	s.password = password
}

func (s *Student) Show() {
	fmt.Println(s.name, s.password)
}
